rd-api-client/src/main/java/org/rundeck/client/api/RundeckApi.java

```diff
@@ -1085,4 +1107,50 @@
     @Headers("Accept: application/json")
     @POST("plugins/uninstall/{pluginId}")
     Call<ArtifactActionMessage> uninstallPlugin(@Path("pluginId") String pluginId);
+
+    /* Bulk toogle job execution. */
+
+    /**
+     * @see <a href="https://rundeck.org/docs/api/#bulk-toggle-job-execution">API</a>
+     */
+    @Json
+    @Headers("Accept: application/json")
+    @POST("jobs/execution/enable")
+    Call<BulkToogleJobExecutionResponse> bulkEnableJobs(
```

Comment:
Typo in several places "Toogle" instead of "Toggle"

Reply:
sorry i never get that word right :)